cilium-dbg/cmd/bpf_nodeid_list.go

```diff
@@ -35,19 +36,20 @@ var bpfNodeIDListCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		common.RequireRootPrivilege("cilium bpf nodeid list")
 
-		bpfNodeIDList := []nodeID{}
-		parse := func(key *nodemap.NodeKey, val *nodemap.NodeValue) {
+		bpfNodeValueList := []nodeID{}
+		parse := func(key *nodemap.NodeKey, val *nodemap.NodeValueV2) {
 			address := key.IP.String()
 			if key.Family == bpf.EndpointKeyIPv4 {
 				address = net.IP(key.IP[:net.IPv4len]).String()
 			}
-			bpfNodeIDList = append(bpfNodeIDList, nodeID{
+			bpfNodeValueList = append(bpfNodeValueList, nodeID{
 				ID:      val.NodeID,
 				Address: address,
+				SPI:     val.SPI,
 			})
 		}
 
-		nodeMap, err := nodemap.LoadNodeMap()
+		nodeMap, err := nodemap.LoadNodeMapV2()
```

Comment:
I'll echo @pchaigno that this command should attempt to dump both maps until the old one is no longer supported.

Reply:
I don't think I said that :sweat_smile: 

I've opened a PR to dump the two raw maps (via bpftool), but I don't think it would bring much to have this command dump and pretty print entries from both maps. It could even be confusing during investigations.

Reply:
Yeah, we don't want the CLI to dump both maps. Rather, we want bugtool to dump the bytes of both maps during systemdump.

However, this isn't occurring as of today see: https://github.com/cilium/cilium/pull/31431#discussion_r1542124198

I'll open a PR which dumps both maps in our systemdumps after this PR.

Reply:
Oh, sorry, I think Paul opened this PR already  :)